app/code/community/Ebizmarts/MailChimp/Model/Api/Products.php

```diff
@@ -783,10 +783,11 @@ public function getParentImageUrl($childId, $magentoStoreId)
      */
     protected function getProductUrl($product, $magentoStoreId)
     {
-        $oldStoreId = Mage::app()->getStore()->getId();
-        Mage::app()->setCurrentStore($magentoStoreId);
+        $helper = $this->getMailChimpHelper();
```

Comment:
Is it necessary to do this or is it already done in the createBatchJson function?